Декоративные элементы всех страниц

Kohize opened the pull request:
---
:mortar_board: [Декоративные элементы всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2019485/tasks/10)

:boom: https://htmlacademy-htmlcss.github.io/2019485-sedona-34/8/

Kohize committed:
.socials and .advantages-cards refactor

Kohize committed:
added divider.svg

Kohize committed:
text refactor for .advantages-list

Kohize committed:
background color tweak for .advantages-list

Kohize committed:
removed "fill" property from <svg>

Kohize committed:
<navigation> refactor, add interacting styles

Kohize committed:
added divider image for .promo-container

Kohize committed:
interaction styles for buttons and pagination

Kohize committed:
footer refactor for catalog.html

Kohize committed:
breadcrumbs refactor in catalog.html

Kohize committed:
small tweaks

Kohize committed:
added font for buttons

AlSudar commented:
Это предпоследняя тема, осталось совсем немного. Давай поправим несколько элементов:
1. На макете шапка слегка залазит на следующий блок, реализуй это в своём коде (при помощи position, z-index, и отрицательного отступа)
2. advantages-list - здесь можно создать для картинок с фоновыми изображениями отдельный класс (например .card-icon), в нём перечислить все повторяющиеся свойства (ширина, высота, отступ снизу), и фоновую картинку задавать в втором классе (лучше её масшатибровать при помощи bg-size). Так мы уменьшим дублирование кода, и в будущем такие картинки будет создавать проще
Должно получиться что-то вроде: `<div class="card-icon card-icon-food"></div>`
3.  Для .filter-container ты загрузил не ту картинку
4. В этот блок уже нужно добавить элементы справа, нет?
5. Стилизуй элемент .select-pagination
6. Картинки с звёздочками лучше сделать так:
Создать элемент (например  hotel-stars-item) и задать ему bg в виде одной звёздочки. И добавлять этот элемент в блок с звёздочками (hotel-stars) столько раз, сколько звёздочек должно быть. 

Пиши, если тебе нужно будет в чём-то помощь

Kohize committed:
tweaks for pull request

added margin for logo in <header>
.card-icon styles refactor
changed bg-image for .filter-container
styled .select-pagination
refactor for .hotel-rate

Kohize committed:
button refactor in .filter-container
